src/main/java/nugget/Ui.java

```diff
@@ -0,0 +1,69 @@
+package nugget;
+
+import nugget.Task;
+
+import java.util.ArrayList;
+
+public class Ui {
+
+    public void line() {
```

Comment:
For this part, I think it would be good if you use printLine() instead of print() to make it in the form of an action